content/guides/2.library_creation.md

```diff
@@ -9,28 +9,27 @@ fullpath: /guides/library_creation
 # Introduction
 This guide will help you to create a library in ABS. This guide assumes you have correctly configured your installation.
 
-Screenshots are from an install of v2.2.0 in Docker, but remain mostly unchanged in v2.3.3.
+NOTE: If you are using docker, make sure your mounts are set up correctly. If they are not set up correctly, you will lose data at some point in the future. The easiest way to check if they are set up correctly is to add a book to the library any ensure that it shows up in your file system. 
+
+Still confused about Docker? Check out [this FAQ](/faq#im-still-confused-about-what-docker-and-containers-are-and-how-they-work)
 
 # Setting up a library
 
 ABS supports two kinds of libraries: book and podcast. Both kinds of library allow individual items to be added to the library. A podcast library can also download new episodes from an RSS feed automatically.
-To create a library file in ABS:
+To create a library in ABS:
 * Create a library directory for either [audiobooks](https://www.audiobookshelf.org/docs#book-structure) or [podcasts](https://www.audiobookshelf.org/docs#podcast-structure)
-* Go to (1) Settings -> (2) Library -> (3) Add Library. This installation already has two libraries set up (named "Audible" and "Podcasts")
-![Add library](/guides/library_setup/add_library.png)
-* Set library paths. You can either type the path manually or Browse for Folder. If you have multiple directories where books/podcasts are stored, you can add multiple paths to the library. NOTE: There are issues with the "Browse for Folder" button on some systems, so you may need to enter the path manually if the button is not working.
+* Go to (1) Settings -> (2) Library -> (3) Add Library.
+![Add library](/guides/library_setup/add_library.jpg)
+* Set library paths. You can either type the path manually or Browse for Folder. If you have multiple directories where books/podcasts are stored, you can add multiple paths to the library. You can have multiple paths in the same library. If you are using docker, you should be using the path within the docker container, not the path in your normal file system. NOTE: There are issues with the "Browse for Folder" button on some systems, so you may need to enter the path manually if the button is not working.
 ![Add paths](/guides/library_setup/initial_library_creation.gif)
-* Modify library settings (you can edit these later)
-![Edit settings](/guides/library_setup/details_4.png)
+* Modify library settings (you can set this up now or edit the settings later)
+![Edit settings](/guides/library_setup/settings.jpg)
 * You can also set a schedule to rescan the library for new books or changes. This periodic scan is disabled by default.
```

Comment:
```suggestion
* You can also set a schedule to rescan the library for new books or changes. This periodic scan is disabled by default. NOTE: This should not be necessary if the watcher is enabled and working because changes to your folders are detected and scanned automatically.
```